rpc/legacyrpc/methods.go

```diff
@@ -477,6 +479,37 @@ func createMultiSig(s *Server, icmd interface{}) (interface{}, error) {
 	}, nil
 }
 
+func createVotingAccount(s *Server, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrjson.CreateVotingAccountCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, errUnloadedWallet
+	}
+
+	name := cmd.Name
+	if len(name) == 0 {
+		return nil, rpcErrorf(dcrjson.ErrRPCInvalidParameter, "empty name")
```

Comment:
validate the account name here using [udb.ValidateAccountName](https://godoc.org/github.com/decred/dcrwallet/wallet/udb#ValidateAccountName)